README.md

```diff
@@ -7,3 +7,216 @@ Status](https://github.com/maltezfaria/DataFlowTasks.jl/workflows/CI/badge.svg)]
+        end
+    end
+
+    # Construct the factorized object
+    return Cholesky(A,'U',zero(LinearAlgebra.BlasInt))
+end
+```
+
+In order to parallelize the code with DataFlowTasks, it will only be necessary to wrap function calls within a `@dspawn`, and add a synchronization point at the end. The parallelized code will be :
+
+```julia
+using DataFlowTasks
+using LinearAlgebra
+function cholesky_tiled!(A, ts)
+    m,n = size(A)
+    n%ts != 0 && error("Tilesize doesn't fit the matrix")
+    tn = n÷ts
+
+    Atile = [view(A, tilerange(ti, ts), tilerange(tj, ts)) for ti in 1:tn, tj in 1:tn]
+
+    for ti in 1:tn
+        # Diagonal cholesky serial factorization (I)
+        @dspawn cholesky!(@RW(Atile[ti,ti])) label="chol ($ti,$ti)"
+
+
+        # Left blocks update (II)
+        L = adjoint(UpperTriangular(Atile[ti,ti]))
+        for tj in ti+1:tn
+            @dspawn ldiv!(@R(L), @RW(Atile[ti,tj])) label="ldiv ($ti,$tj)"
+        end
+
+        # Submatrix update (III)
+        for tj in ti+1:tn
+            for tk in tj:tn
+                Ajk = Atile[tj,tk]
+                Aik = Atile[ti,tk]
+                Aji = adjoint(Atile[ti,tj])
+                @dspawn mul!(@RW(Ajk), @R(Aji), @R(Aik), -1, 1) label="schur ($tj,$tk)"
+            end
+        end
+    end
+    # Construct the factorized object
+    r = @dspawn Cholesky(A,'U',zero(LinearAlgebra.BlasInt))
+    return fetch(r)
+end
+```
+
+If you run this code, you'll see `memory_overlap` warning on the fact that there's no method to evaluate the memory overlaping between an `Adjoint` and a `SubArray`. So we have to implement this method :
+
+```julia
+using DataFlowTasks: memory_overlap
+memory_overlap(U::Adjoint,A) = memory_overlap(U.parent,A)
+memory_overlap(U,L::Adjoint) = memory_overlap(L,U)
+```
+
+The code below shows how to use this `cholesky_tiled!` function, how to profile the program and get the most information from the visualization. 
+
+```julia
+import DataFlowTask as DFT
+
+# DataFlowTasks environnement setup
+DFT.reset!()
+DFT.enable_log()
+DFT.setscheduler!(DFT.JuliaScheduler(50))
+
+# Context
+n  = 4096
+ts = 512
+A = rand(n, n)
+A = (A + adjoint(A))/2
+A = A + n*I
+
+# Compilation
+cholesky_tiled!(copy(A), ts)
+
+# Reset environnement
+DFT.resetlogger!()
+GC.gc()
+
+# Real work to be analysed
+F = cholesky_tiled!(A ,ts)
+@info err  = norm(F.L*F.U-A,Inf)/max(norm(A),norm(F.L*F.U))
+
+# Parallel Trace Plot
+DFT.plot(categories=["chol", "ldiv", "schur"])
+
+# Dag Plot
+DFT.dagplot()
+```
+
+## Profiling
+
+The next results are obtained with a slightly modified version of the function presented above. We used the `LoopVectorization` for the serial cholesky (I), `TriangularSolve`'s `ldiv!` and `Octavian`'s `matmul_serial!`.
+
+The package `TiledFactorization` contains this implementation, you can use it with the following lines :
+
+```julia
+import Pkg
+Pkg.add("https://github.com/maltezfaria/TiledFactorization.git")
+using TiledFactorization: cholesky!
+```
```

Comment:
I think profiling should be illustrated on the example developed above. After all, we're not showing off with high-performance here, we are merely presenting a use-case.

IMO, only the performance plots should be coming from the `TiledFactorization` package.